corehq/apps/export/templates/export/partials/delete_bulk_notice.html

```diff
@@ -0,0 +1,45 @@
+{% load i18n %}
+<div data-bind="visible: bulkExportDownloadCount()">
+  <div style="float:right;" class="tooltip-bulkDelete" data-container=".tooltip-bulkDelete"
```

Comment:
if this just needs to float, you can use the bootstrap utility class `pull-right` :)